lib/gridfs-stream/upload.js

```diff
@@ -447,3 +486,15 @@ function writeRemnant(_this, callback) {
     checkDone(_this);
   });
 }
+
+/**
+ * @ignore
+ */
+
+function checkAborted(_this, callback) {
```

Comment:
In fact there is still a check missing for the call to `checkDone()` in the callback of `chunks.insert` in `writeRemnant`.

I think it would be more robust and simpler to just check before all inserts (the line before the insert would help verifications), so add it in `checkDone()` too.